src/print.jl

```diff
@@ -819,7 +819,7 @@ function con_str(mode, c::GenericRangeConstraint, sym)
     out_str
 end
 # Backwards compatability shim
-conToStr(c::GenericRangeConstraint) = con_str(REPLMode,c)
+constr(c::GenericRangeConstraint) = con_str(REPLMode,c)
```

Comment:
If these are actually "backwards compatability shims" then we should just be able to remove them by now.